putty.txt

```diff
@@ -10,6 +11,8 @@ URLDownload = https://the.earth.li/~sgtatham/putty/0.70/w32/putty-0.70-installer
 SHA1 = 2031fc9b6a972f5c3f3e506987143dfc1f7214e0
 CDPath = none
 SizeBytes = 2942464
+Languages=0409|0407|040A|040C|0410|0413|0415|0418|0419|041F|0422|0804|0813 \\English|German|French|Italian|Dutch|Polish|Romanian|Russian|Turkish|Ukranian|Chinese|Dutch-Belgium
```

Comment:
Why are there putty changes in this PR?

Reply:
that was meant to be seperate. I'll fix the rest.